### PR TITLE
Restitued Orbit File Support 

### PR DESCRIPTION
**Pull request by collinss-jpl**
This PR updates the stage_orbit_file.py tool to support querying/downloading of Restitued Orbit File (ROEORB) in addition to the current support for Precise Orbit (POEORB) files. Selection of the orbit file type is accomplished via a --orbit-type argument added to stage_orbit_file.py.

The daac_data_subscriber.py script has also been updated to fall back to querying for RESORB files if an initial query for POEORB files returns no results.

In the course of testing this branch, I discovered that ASF DAAC had started serving s3 URL's in addition to the HTTPS locations, so the subscriber script has been updated to store the s3 URL's, if provided, in elasticsearch along side the HTTPS address. When downloading, if HTTPS is present it is still prioritized over the S3 location.

Lastly, the branch ups the EBS size for the /data partition on SLC download workers to 100 gigs, as the previous size was shown to quickly fill up for hourly query windows.

### CODE REVIEW

File: tools/stage_orbit_file.py [hunk 1]
```diff
@@ -179,42 +200,46 @@ def construct_orbit_file_query(mission_id, safe_start_time, safe_stop_time):
...
 
-    # Pad the start/stop times by a day on each side to ensure we can find
+    # Pad the start/stop times on each side to ensure we can find
     # a corresponding Orbit file that encompasses the SAFE time range
-    query_start_date = safe_start_date - timedelta(days=1)
-    query_stop_date = safe_stop_date + timedelta(days=1)
+    if orbit_type == ORBIT_TYPE_POE:
+        query_delta = timedelta(days=1)
+    else:
+        query_delta = timedelta(hours=3)
```
[the remark this answers is not in the document]
Comment (collinss-jpl) [reply tools/stage_orbit_file.py:208]: The precise and restitued orbit files are produces at different cadences, as well as cover different amounts of time per-file. From the ADT product spec describing the files:

▪	Precise Orbit Ephemerides (POE): delivered within 20 days after the S1-A/B data acquisition and exhibiting less than 5 cm accuracy.
▪	Restituted Orbit Ephemerides (ROE): delivered within 3 hours after the S1-A/B acquisition and exhibiting less than 10 cm accuracy. 

Basically we can expect the precise files to be available for historical processing, and restituted to be available in time for keep-up processing. Since the precise files cover a longer time span, we have to cast a "wider net" when looking for a file that overlaps completely with the input SLC file, hence the padding by a day on each side, versus only 3 hours for restituted files.

[the remark this answers is not in the document]
Comment (riverma) [reply tools/stage_orbit_file.py:208]: Thanks for that explanation @collinss-jpl. Got it 👍
